openshift-gitops/infra/base/projects.yaml

```diff
@@ -3,30 +3,30 @@ apiVersion: argoproj.io/v1alpha1
 kind: AppProject
 metadata:
   name: dataplane
-  namespace: openshift-gitops
   annotations:
-    apps: apps.dp.kni.syseng.devcluster.openshift.com
+    apps: dataplane
```

Comment:
The apps: will be `app.dp.kni.syseng.devcluster.openshift.com` if the dataplane server is `https://api.dp.kni.syseng.devcluster.openshift.com:6443`

Reply:
can you elaborate on this requirement?

Reply:
This is being used to patch a resource in post-install.

Reply:
why can't a more generic name be used so that it does not depend on a specific server url?

Reply:
can you specify where is exactly used? I'd like to find a way to perform the patch without requireing the use of the server url, otherwise will complicate the portability of this solution

Reply:
You will find the code in gateway/clustering.yaml
`
# patch rool url and redirect url for keycloak client
    if ! argocd app patch-resource dp-prereqs --kind KeycloakClient --resource-name kong-demo-client --namespace=keycloak --patch "{\"spec\":{\"client\":{\"rootUrl\": \"http://kong-kong-proxy-kong.$(oc get appprojects -n openshift-gitops dataplane -ojsonpath='{.metadata.annotations.apps}')/\",\"redirectUris\":[\"http://kong-kong-proxy-kong.$(oc get appprojects -n openshift-gitops dataplane -ojsonpath='{.metadata.annotations.apps}')/*\"]}}}" --patch-type 'application/merge-patch+json'; then
      echo "keycloak client not found"
      exit 1
    fi
`